pkg/tuf/common.go

```diff
@@ -1,24 +1,19 @@
-// Most of the helper functions are adapted from github.com/theupdateframework/notary
-//
-// Figure out the proper way of making sure we are respecting the licensing from Notary
-// While we are also vendoring Notary directly (see LICENSE in vendor/github.com/theupdateframework/notary/LICENSE),
-// copying unexported functions could fall under different licensing, so we need to make sure.
-
 package tuf
 
 import (
 	"os"
 	"path/filepath"
 	"runtime"
 
-	"github.com/docker/distribution/reference"
-	"github.com/docker/docker/registry"
+	"github.com/theupdateframework/notary/tuf/data"
 )
 
 const (
-	dockerConfigDir = ".docker"
+	dockerConfigDir  = ".docker"
+	releasesRoleName = data.RoleName("targets/releases")
 )
 
+// DefaultTrustDir returns where the Signy trust data lives
 func DefaultTrustDir() string {
 	homeEnvPath := os.Getenv("HOME")
```

Comment:
FYI, you can use [os.UserHomeDir()](https://golang.org/pkg/os/#UserHomeDir) to accomplish this task. Though I am am wondering if eventually signy data directory should be be configurable so that it can be stored in the CNAB tool's home directory, e.g. in ~/.porter/signy or ~/.duffle/signy?